db/migrate/20160609080700_add_clp_datamodel.rb

```diff
@@ -0,0 +1,19 @@
+class AddClpDatamodel < ActiveRecord::Migration
+  def up
+    create_table :landingpages do |t|
+      t.column :community_id, :integer, null: false
+      t.column :version, :integer, null: false
+      t.column :released, :datetime
+      t.column :content, :text, limit: 16.megabytes - 1, null: false
```

Comment:
To my understanding, `16.megabytes - 1` is the default, so it's not needed here. If you want to keep it here for the sake of explicity, then 👍